src/main/java/clients/symphony/api/datafeeds/DatafeedClientV1.java

```diff
@@ -106,4 +119,30 @@ private void writeDatafeedIdToDisk(SymConfig config, String datafeedId) {
             return datafeedEvents;
         }
     }
+
+
+    /**
+     * {@inheritDoc}
```

Comment:
Might be worth documenting in the Javadoc that those methods are not supported for DFv1

Reply:
I updated the javadoc.